packages/grafana-ui/src/components/Logs/LogRow.tsx

```diff
@@ -157,25 +182,29 @@ class UnThemedLogRow extends PureComponent<Props, State> {
       theme,
       getFieldLinks,
     } = this.props;
-    const { showDetails, showContext, hasHoverBackground } = this.state;
+    const { showDetails, showContext, hasHoverBackground, hasError, errorTooltip } = this.state;
     const style = getLogRowStyles(theme, row.logLevel);
     const styles = getStyles(theme);
-    const hoverBackground = cx(style.logsRow, { [styles.hoverBackground]: hasHoverBackground });
+    const logRowBackground = cx(style.logsRow, {
+      [styles.hoverBackground]: hasHoverBackground,
+      [styles.erroredLog]: hasError,
+    });
 
     return (
       <>
-        <tr
-          className={hoverBackground}
-          onMouseEnter={this.addHoverBackground}
-          onMouseLeave={this.clearHoverBackground}
-          onClick={this.toggleDetails}
-        >
+        <tr className={logRowBackground} onClick={this.toggleDetails}>
```

Comment:
I have removed `onMouseEnter` and `onMouseLeave` as it was breaking the tooltip - tooltip often wasn't showing. We are still passing these methods to Log Details so if we move with cursor to log details, log row is going to get the background color as well.